app/styles/modules/_button-row.scss

```diff
@@ -1,88 +1,89 @@
-.button-row {
+.button,
+.settings-button,
+button {
+  @include font();
+  align-items: center;
+  background: $button-background-color-default;
+  border: 0;
+  border-radius: $button-border-radius-default;
+  color: $button-text-color-default;
+  cursor: pointer;
+  display: inline-flex;
+  font-size: $button-font-size-default;
+  font-weight: $button-font-weight-default;
+  height: $button-height-default;
+  justify-content: center;
+  letter-spacing: 0.2px;
+  padding: 0 8px;
+  transition-duration: $short-transition;
+  transition-property: background-color;
+  // it is very easy to accidentally select the text when clicking
+  -webkit-touch-callout: none;
+  // user select not supported by autoprefixer
+  // https://github.com/postcss/autoprefixer/issues/167
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+  width: 100%;
 
-  @include respond-to('reasonableUI') {
-    margin: 20px 0;
+
+  &:hover {
+    background: $button-background-color-default-hover;
   }
 
-  @include respond-to('trustedUI') {
-    margin: 10px 0;
+  &:active {
+    background: $button-background-color-default-active;
   }
 
-  .button {
-    display: inline-block;
-    text-decoration: none;
+  &:focus {
+    box-shadow: 0 0 0 1px #0a84ff inset, 0 0 0 1px #0a84ff, 0 0 0 4px rgba(10, 132, 255, 0.3);
   }
 
-  .button,
-  button {
-    @include font();
-    background: $button-background-color;
-    border: 1px solid $button-border-default-color;
-    color: $message-text-color;
-    cursor: pointer;
-    transition-duration: $short-transition;
-    transition-property: background-color;
-    width: 100%;
-
-    @include respond-to('big') {
-      border-radius: $big-border-radius;
-      font-size: $large-font;
-      height: 65px;
-      padding: 15px 0;
+  &[type='submit'],
+  &.primary-button {
```

Comment:
Did you have to add the `-button` suffix to avoid conflicts? Seems like we could drop it.

Reply:
Precisely, it started with `warning`. There were both `warning` messages and buttons, applying styles to one updated the styles for the other and I didn't want that. So I started with `warning-button`, then added the suffix to the others to stay consistent and because `primary` and `secondary` can be applied to anything really.